jaxley/connect.py

```diff
@@ -44,33 +44,52 @@ def fully_connect(
     pre_cell_view: "View",
```

Comment:
should we rename `fully_connect` to `fully_connect_cells`? Might be less confusing, since otherwise connections are made comp 2 comp